aa_stripe/management/commands/sync_all_cards_for_all_customers.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+import stripe
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from aa_stripe.models import StripeCard, StripeCustomer
+from aa_stripe.settings import stripe_settings
+
+
+class Command(BaseCommand):
+    help = "Sync all cards for all customers with Stripe API"
+
+    def add_arguments(self, parser):
+        user_ids = get_user_model().objects.values_list("id")
+        min_user_id, max_user_id = min(user_ids)[0], max(user_ids)[0]
```

Comment:
You're loading IDs of ALL users from DB to the app just to make min & max on them... hard to imagine a less efficient way to do it. If the number of users in the system is 1mln, you'll send 1mln ids to the app just to take two values out of them.

First of all, it is not necessary at all - if there is a parameter given, you add __lte or __gte to the filter and that's all. You don't need to check what are the extreme values. But if you really want, you could do just .objects.order_by("id").first().id / objects.order_by("-id").first()

Another thing is that you're iterating over Account. Why? You want to update StripeCustomers, so you should iterate over them. You can do 

processed_users_set = set()
for sc in StripeCustomer.objects.filter(is_active=True).order_by("-id"):
   if sc.user_id in processed_users_set:
      continue
   ....
   processed_users_set.add(sc.user_id)

this way you are doing way less queries and still updating just the latest customer for a given user.